app/javascript/gobierto_data/webapp/components/commons/InfoTab.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <div class="pure-g">
+    <div class="pure-u-17-24 gobierto-data-summary-header">
+      <div
+        id="gobierto-data-summary-header"
+        class="gobierto-data-summary-header-description"
+        v-html="compiledHTMLMarkdown"
+      />
+    </div>
+    <div class="pure-u-7-24">
+      <div
+        class="gobierto-data-summary-header-btns"
+      >
+        <template v-if="moreThanOneFormat">
+          <DownloadButton
+            :array-formats="arrayFormats"
+            class="arrow-top modal-left"
+          />
+        </template>
+        <template v-else>
+          <DownloadLink
+            :editor="false"
+            :array-formats="arrayFormats"
+          />
+        </template>
+        <router-link
+          :to="`/datos/${$route.params.id}/${tabs[1]}`"
+          class="gobierto-data-btn-preview"
+        >
+          <Button
+            :text="labelPreview"
+            icon="table"
+            color="rgba(var(--color-base)"
+            icon-color="rgba(var(--color-base-string), .5)"
+            class="gobierto-data-btn-download-data "
+            background="#fff"
+          />
+        </router-link>
+      </div>
+      <InfoBlockText
+        v-if="dateUpdated"
+        icon="clock"
+        :icon-size="'12px'"
+        :icon-color="'#666'"
+        :label="labelUpdated"
+        :text="dateUpdated | convertDate"
+      />
+      <InfoBlockText
+        v-if="frequencyDataset"
+        icon="calendar"
+        :icon-size="'12px'"
+        :icon-color="'#666'"
+        :label="labelFrequency"
+        :text="frequencyDataset"
+      />
+      <InfoBlockText
+        v-if="categoryDataset"
+        icon="tag"
+        :icon-size="'12px'"
+        :icon-color="'#666'"
+        :label="labelSubject"
+        :text="categoryDataset"
+      />
+      <InfoBlockText
+        v-if="hasDatasetSource"
+        icon="building"
+        :icon-size="'12px'"
+        :icon-color="'#666'"
+        :label="labelSource"
+        :text="sourceDatasetText"
+        :url="sourceDatasetUrl"
+      />
+      <InfoBlockText
+        v-if="hasDatasetLicense"
+        icon="certificate"
+        :icon-size="'12px'"
+        :icon-color="'#666'"
+        :label="labelLicense"
+        :text="licenseDatasetText"
+        :url="licenseDatasetUrl"
+      />
+    </div>
+  </div>
+</template>
+<script>
+import { date } from "lib/vue/filters"
+import InfoBlockText from "./../commons/InfoBlockText.vue";
+import DownloadButton from "./../commons/DownloadButton.vue";
+import DownloadLink from "./../commons/DownloadLink.vue";
+import Button from "./../commons/Button.vue";
+import { tabs } from "../../../lib/router";
+//Parse markdown to HTML
+const marked = require('marked');
```

Comment:
Why `require`?

Reply:
Because this version of the marked need require